node/tests/TestUtilities.ts

```diff
@@ -13,18 +13,13 @@ import {
     GlideClient,
     GlideClusterClient,
     InsertPosition,
-    Logger,
     ProtocolVersion,
     ReturnType,
     Transaction,
 } from "..";
 import { checkIfServerVersionLessThan } from "./SharedTests";
 import { LPosOptions } from "../build-ts/src/command-options/LPosOptions";
 
-beforeAll(() => {
-    Logger.init("info");
-});
```

Comment:
Why remove this?

Reply:
It produces too much noise to the test output and makes it harder to read.

Reply:
Maybe we could change the log level instead? I'm not totally against removing this, but I'm a little concerned if it loses us valuable debug information.

Reply:
Warnings and errors are still printed with the default level.
This also increases overall performance of the test execution, because it eliminates a lot of stdout IO and makes test output readable.